package.json

```diff
@@ -61,46 +61,46 @@
   },
   "dependencies": {
     "ansi-colors": "^4.1.3",
-    "body-parser": "^1.20.1",
-    "browser-sync": "^2.27.11",
+    "body-parser": "^1.20.2",
+    "browser-sync": "^2.29.3",
     "chokidar": "^3.5.3",
     "cookie-parser": "^1.4.6",
     "cross-spawn": "^7.0.3",
-    "csrf-csrf": "^2.2.4",
+    "csrf-csrf": "^2.3.0",
     "del": "^6.1.1",
-    "dotenv": "^16.0.3",
+    "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "express-session": "^1.17.3",
-    "fs-extra": "^11.1.0",
+    "fs-extra": "^11.1.1",
     "govuk-frontend": "4.7.0",
-    "inquirer": "^8.2.0",
+    "inquirer": "^8.2.6",
     "lodash": "^4.17.21",
-    "marked": "^4.2.5",
+    "marked": "^6.0.0",
```

Comment:
Not sure how widely `marked` is used, but the recent releases deprecate and drop lots of features (they're now extensions)

Reply:
Yes, perhaps I was a little overzealous with marked, I'll change it to the latest 4.x.x